test/test_transforms_v2_functional.py

```diff
@@ -687,7 +687,9 @@ def _compute_expected_bbox(bbox, top_, left_, height_, width_, size_):
     if format != datapoints.BoundingBoxFormat.XYXY:
         in_boxes = convert_format_bounding_boxes(in_boxes, datapoints.BoundingBoxFormat.XYXY, format)
 
-    output_boxes, output_canvas_size = F.resized_crop_bounding_boxes(in_boxes, format, top, left, height, width, size)
+    output_boxes, output_canvas_size = F.resized_crop_bounding_boxes(
+        in_boxes.as_subclass(torch.Tensor), format, top, left, height, width, size
```

Comment:
This (and similar changes below) was needed because `in_boxes` is now still a BBox instance, and `resized_crop_bounding_boxes` expects a tensor (there is an error saying something like "if you pass a bbox, don't pass the format").